ArduPlane/GCS_Mavlink.cpp

```diff
@@ -1405,7 +1418,63 @@ void GCS_MAVLINK_Plane::handle_set_position_target_global_int(const mavlink_mess
             if (msg_valid) {
                 handle_change_alt_request(cmd);
             }
-        } // end if alt_mask       
+        } // end if alt_mask
+
+        // guided path following
+        if (((pos_target.type_mask | POSITION_TARGET_TYPEMASK_LAST_BYTE) ^ 0xFFFF) == path_mask) {
+            cmd.content.location.lat = pos_target.lat_int;
+            cmd.content.location.lng = pos_target.lon_int;
+
+            cmd.content.location.alt = pos_target.alt * 100;
+            cmd.content.location.relative_alt = false;
+            cmd.content.location.terrain_alt = false;
+            switch (pos_target.coordinate_frame) 
```

Comment:
Try `mavlink_coordinate_frame_to_location_alt_frame`

i.e. extract as much useful stuff from our `COMMAND_INT` / `COMMAND_LONG` parsing...

Reply:
Rebase on top of #28479 once that's merged.